application/common/components/Emailer.php

```diff
@@ -24,7 +60,7 @@ class Emailer extends Component
      * @param string $subject The subject.
      * @param string $htmlBody The email body (as an HTML string).
      */
-    public function email(string $toAddress, string $subject, string $htmlBody)
+    protected function email(string $toAddress, string $subject, string $htmlBody)
```

Comment:
Since this class is purely for use from within ID Broker, and it seemed likely that we will always be providing HTML email bodies, I didn't see a reason to allow us (from within ID Broker) to specify both an HTML body and a text body.

I did wonder about whether the Email Service would automatically generate a text body from the HTML body (if no text body was provided) or whether I should do that here. Would you like me to change it to something like the following?
```php
$this->getEmailServiceClient()->email([
    'to_address' => $toAddress,
    'subject' => $subject,
    'html_body' => $htmlBody,
    'text_body' => strip_tags($htmlBody),
]);
```